libraries/redshop/tags/sections/manufacturerdetail.php

```diff
@@ -0,0 +1,224 @@
+<?php
+/**
+ * @package     RedSHOP.Library
+ * @subpackage  Tags
+ *
+ * @copyright   Copyright (C) 2008 - 2020 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE
+ */
+
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Factory;
+
+defined('_JEXEC') || die;
+
+/**
+ * Tags replacer abstract class
+ *
+ * @since  3.0
+ */
+class RedshopTagsSectionsManufacturerDetail extends RedshopTagsAbstract
+{
+	/**
+	 * @var    integer
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public $itemId;
+
+	/**
+	 * Init
+	 *
+	 * @return  mixed
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function init()
+	{
+		$this->itemId = Factory::getApplication()->input->getInt('Itemid', 0);
+	}
+
+	/**
+	 * Execute replace
+	 *
+	 * @return  string
+	 *
+	 * @since   __DEPLOY_VERSION
+	 */
+	public function replace()
+	{
+		$db        = Factory::getDbo();
+		$pageTitle = Text::_('COM_REDSHOP_MANUFACTURER_DETAIL');
+
+		if ($this->data['pageHeadingTag'] != '') {
+			$pageTitle = $this->data['pageHeadingTag'];
+		}
+
+		$this->template = RedshopLayoutHelper::render(
+				'tags.common.pageheading',
+				[
+					'params'         => $this->data['params'],
+					'pageheading'    => $db->escape($this->data['params']->get('page_title')),
+					'pageHeadingTag' => $pageTitle,
+					'class'          => 'manufacturer-detail'
+				],
+				'',
+				$this->optionLayout
+			) . $this->template;
+
+		$row            = !empty($this->data['detail']) ? $this->data['detail'][0] : null;
```

Comment:
@hungnguyenckc I think condition check is `$row = $this->data['detail'][0] ?? null;` more suitable